src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ProductReview.xml

```diff
@@ -20,31 +20,38 @@
             <group>product_review:read</group>
         </attribute>
         <attribute name="createdAt">
-           <group>product_review:read</group>
-       </attribute>
+            <group>product_review:read</group>
+            <group>shop:product:read</group>
+        </attribute>
         <attribute name="updatedAt">
-           <group>product_review:read</group>
-       </attribute>
-       <attribute name="title">
-           <group>product_review:read</group>
-           <group>product_review:update</group>
-       </attribute>
-       <attribute name="rating">
-           <group>product_review:read</group>
-           <group>product_review:update</group>
-       </attribute>
-       <attribute name="comment">
-           <group>product_review:read</group>
-           <group>product_review:update</group>
-       </attribute>
-       <attribute name="author">
-           <group>product_review:read</group>
-       </attribute>
-       <attribute name="status">
-           <group>product_review:read</group>
-       </attribute>
-       <attribute name="reviewSubject">
-           <group>product_review:read</group>
-       </attribute>
+            <group>product_review:read</group>
```

Comment:
I am not sure when we added this serialization, but changes like this should be mentioned in upgrade file

Reply:
@AdamKasp If I see correctly, the only change here is adding new group for shop read, so there is no need for mentioning in UPGRADE file

Reply:
in this pr i just changed 3 spaces into 4, for some reason it was 3 since this file was made